src/pool/poolClient.ts

```diff
@@ -278,69 +302,59 @@ function joinPoolState(
     address: poolState.address,
     totalPoolSize: totalPoolSize.toString(),
     l1Token: poolState.l1Token,
+    lpToken: poolState.lpToken,
     liquidReserves: poolState.liquidReserves.toString(),
-    pendingReserves: poolState.pendingReserves.toString(),
     exchangeRateCurrent: poolState.exchangeRateCurrent.toString(),
     exchangeRatePrevious: poolState.exchangeRatePrevious.toString(),
     estimatedApy,
     estimatedApr,
     blocksElapsed,
     secondsElapsed,
-    liquidityUtilizationCurrent: poolState.liquidityUtilizationCurrent.toString(),
     projectedApr,
     utilizedReserves: poolState.utilizedReserves.toString(),
   };
 }
 export class ReadPoolClient {
   private poolState: PoolState;
-  private multicall: uma.Multicall2;
-  private contract: uma.clients.bridgePool.Instance;
-  private batchRead: BatchReadWithErrorsType;
-  constructor(private address: string, private provider: Provider, private multicallAddress: string) {
-    this.multicall = new Multicall2(this.multicallAddress, this.provider);
-    this.contract = bridgePool.connect(address, this.provider);
-    this.batchRead = BatchReadWithErrors(this.multicall)(this.contract);
-    this.poolState = new PoolState(this.batchRead, this.contract, this.address);
-  }
-  public async read(latestBlock: number) {
-    return this.poolState.read(latestBlock);
+  private contract: hubPool.Instance;
+  constructor(private address: string, private provider: Provider) {
+    this.contract = hubPool.connect(address, this.provider);
+    this.poolState = new PoolState(this.contract, this.address);
+  }
+  public async read(tokenAddress:string, latestBlock: number) {
+    return this.poolState.read(tokenAddress, latestBlock);
   }
 }
 export function validateWithdraw(pool: Pool, user: User, lpTokenAmount: BigNumberish) {
   const l1TokensToReturn = BigNumber.from(lpTokenAmount).mul(pool.exchangeRateCurrent).div(fixedPointAdjustment);
   assert(BigNumber.from(l1TokensToReturn).gt("0"), "Must withdraw amount greater than 0");
-  assert(
-    BigNumber.from(pool.liquidReserves).gte(l1TokensToReturn.add(pool.pendingReserves)),
-    "Utilization too high to remove that amount, try lowering withdraw amount"
-  );
```

Comment:
i removed this because therse no pending reserves, but this means theres no client side error if a user is trying to remove more than the active liquidity